SolarSharp.Interpreter/Security/CrossPlatformPathCanonicalizer.cs

```diff
@@ -15,11 +14,6 @@ namespace SolarSharp.Interpreter.Security
     /// </summary>
     public sealed class CrossPlatformPathCanonicalizer
     {
-        private readonly ConcurrentDictionary<string, CachedPath> _pathCache = new();
-        private readonly TimeSpan _cacheExpiry;
-        private readonly object _cleanupLock = new();
-        private DateTime _lastCleanup = DateTime.UtcNow;
```

Comment:
The cache here is a **lot** of code, and has multiple layers of complexity.  It also introduces a lock.  I'm really not sure that the value here is > the cost so I'm removing to also simplify the fixes here.